cmd/src/server.rs

```diff
@@ -208,7 +208,7 @@ impl TiKVServer {
         validate_and_persist_config(&mut config, true);
         check_system_config(&config);
 
-        tikv_util::set_panic_hook(false, &config.storage.data_dir);
+        tikv_util::set_panic_hook(false, &config.storage.data_dir, &config.panic_log_file);
```

Comment:
Is `panic_log_file` necessary? You can try `stderr` directly. tidb-ansible will redirect it to `tikv_stderr.log`.

Reply:
yes,use io redirect can solve this problem， but I think it's not bad than the origin :),  I just pickup the [ucp challenge](https://github.com/tikv/tikv/issues/7220)